### DISCUSSION TITLE
Strange behavior on chia_full_node connections

### DISCUSSION THREAD

**Discussion by kata32** [Chia-Network/chia-blockchain#5558] [Support]
Strange behavior on chia_full_node connections. The IPs highlighted in green are the correct ones, the IPs highlighted in yellow correspond to my default gateway. What's happening?

![full_node_connections](https://user-images.githubusercontent.com/26146898/118881612-94538f80-b8f3-11eb-9a79-c7c7281bd7f6.png)

**Comment by ghost** [comment]
Hi

Do you run Chia in a Docker container? I see similar things, also lot of FULL_NODE entries using my Docker bridge gateway address.
But all of them have different hights and hashes - so looks like it's working?

**Reply by kata32** [reply]
I don't use doker, it's a physical machine behind a Fortinet firewall. 
I don't know if it's working, there are also (few) right addresses

**Comment by ghost** [comment]
Hi

Try version 1.1.6 - the current latest version. This version has improved the connectivity thing on my end.

**Reply by kata32** [reply]
May be, I'll install it tonight

**Comment by BrandtH22** [comment]
@kata32 , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
